### ISSUE TITLE
Add helper for "TargetRef support matrix"

### ISSUE THREAD

**Issue by slonka** [kumahq/kuma-website#1114]
### Description

This will allow us to be more DRY and automatically updated for new target refs.

This can probably be implemented as a plugin like "tip" - https://github.com/kumahq/kuma-website/blob/b38c85219da0f338dfff718129f464da4130027a/app/_plugins/tags/tip.rb#L1-L0

**Comment by michaelbeaumont** [comment]
Sorry I didn't bring this up earlier but doesn't the list of supported kinds depend on the policy? I.e. doesn't it have to be explicitly updated for each policy anyway?

**Comment by slonka** [comment]
It does but I'm guessing we could have a default of "❌" so that there are no target ref kinds missing from every table - and I think that was @lahabana point.

**Comment by lahabana** [comment]
Yes but you can have a helper with just the list of supported kinds instead of all kinds.
E.g: {%RargetRefTable Top=Mesh,MeshService From=Mesh %}

It would make it simpler for maintaining new targetRefs etc.

**Comment by slonka** [comment]
to get specific, will 

```
{%RargetRefTable Top=Mesh,MeshService From=Mesh %}
```

render:

| TargetRef kind    | top level | to  | from |
|-------------------|-----------|-----|------|
| Mesh              | ✅         | ❌   | ✅    |
| MeshSubset        | ❌         | ❌   | ❌    |
| MeshService       | ✅         | ❌   | ❌    |
| MeshServiceSubset | ❌         | ❌   | ❌    |

or

| TargetRef kind    | top level | to  | from |
|-------------------|-----------|-----|------|
| Mesh              | ✅         | ❌   | ✅    |
| MeshService       | ✅         | ❌   | ❌    |

?

I think the first option is better - that way the table looks the same always and a user will get used to it.

**Comment by lahabana** [comment]
Agreed 1 is better (visual memory will help users too)

**Comment by lahabana** [comment]
Ideally we could use the json schema directly as it's in there

**Comment by lahabana** [comment]
We concluded that this wasn't that helpful in the end
